### PR TITLE
feat: adding decoration.html

### PR DESCRIPTION
**Pull request by LeVHoangduc**
@lenguyen-agilityio @hungnguyenhuu Please help me to check and merge it.

### CODE REVIEW

File: practice-two/src/partials/components/decoration.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<div class="decoration">
+  <div class="decoration--layout">
+    <div class="decoration--top">
+      <div class="decoration--middle"></div>
+    </div>
+    <div class="decoration--bottom"></div>
```
Comment (hungnguyenhuu) [inline practice-two/src/partials/components/decoration.html:6]: This block of code looks weird. Top contains middle and the bottom is the same level with bottom.

Comment (LeVHoangduc) [reply practice-two/src/partials/components/decoration.html]: @hungnguyenhuu i have updated it. Please help me check it again.

File: practice-two/src/partials/components/decoration.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<div class="decoration">
+  <div class="layout">
+    <div class="layout__background">
+      <div class="layout__background__center"></div>
```
Comment (hungnguyenhuu) [inline practice-two/src/partials/components/decoration.html:4]: center? should be layout__background--center, right?

Comment (LeVHoangduc) [reply practice-two/src/partials/components/decoration.html]: i have updated it. Thanks for your advice @hungnguyenhuu

**Pull request merged**